mmselfsup/models/target_generators/clip_generator.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Tuple
+
+import torch
+from mmengine.model import BaseModule
+
+from mmselfsup.registry import MODELS
+from ..utils import build_clip_model
+
+
+@MODELS.register_module()
+class CLIPGenerator(BaseModule):
+    """Get the features and attention from the last layer of CLIP.
+
+    This module is used to generate target features in masked image modeling.
+
+    Args:
+        tokenizer_path (str): The path of the checkpoint of CLIP.
+    """
+
+    def __init__(self, tokenizer_path: str) -> None:
+        super().__init__()
+        self.tokenizer = build_clip_model(torch.load(tokenizer_path), False)
```

Comment:
better to support to load from url
using mmengine.runner.checkpoint.load_checkpoint is better.